tools/azsdk-cli/update-mcp-tools-docs.ps1

```diff
@@ -0,0 +1,125 @@
+<#
+.SYNOPSIS
+    Updates the MCP tools documentation by extracting tool names and descriptions from the Azure SDK CLI.
+
+.DESCRIPTION
+    This script runs the Azure SDK CLI to list all available MCP tools and generates
+    a markdown documentation file (mcp-tools.md) containing the tool names and descriptions
+    in a table format.
+
+.PARAMETER AzsdkExePath
+    Path to the Azure SDK CLI executable (azsdk).
+
+.EXAMPLE
+    ./update-mcp-tools-docs.ps1 -AzsdkExePath "./azsdk"
+#>
+
+param(
+    [Parameter(Mandatory=$true)]
+    [string]$AzsdkExePath
+)
+
+$ErrorActionPreference = "Stop"
+
+function Get-McpVersion {
+    param([string]$exePath)
+    
+    if (-not (Test-Path $exePath)) {
+        throw "Azure SDK CLI executable not found at: $exePath"
+    }
+    
+    $versionOutput = & $exePath --version 2>&1
+    
+    if ($LASTEXITCODE -ne 0) {
+        throw "Failed to run azsdk --version command. Exit code: $LASTEXITCODE"
+    }
+    
+    # Remove the +sha part from version (e.g., "0.5.9+adef6a4419984a5852e4118d05f695b90d007044" -> "0.5.9")
+    $version = $versionOutput.Trim() -replace '\+.*$', ''
```

Comment:
I just put up https://github.com/Azure/azure-sdk-tools/pull/13182 which will remove the sha from the version (I don't think it's helpful to have anyway).